packages/react-widgets/src/DateTimePicker.tsx

```diff
@@ -173,6 +177,140 @@ const defaultProps = {
   formats: {},
 }
 
+export interface DateTimePickerProps extends Omit<WidgetProps, "min" | "max" | "value" | "onChange" | "onSelect"> {
```

Comment:
this should use `DateLocalizationProps<TLocalizer>` i think, which has the correct types for `formats`, check out how Calendar has it

Reply:
Fixed, but I had to keep the Omits for onChange and onSelect